packages/plugin-version/sources/commands/version/check.tsx

```diff
@@ -472,16 +472,13 @@ async function fetchRoot(initialCwd: PortablePath) {
 }
 
 async function fetchChangedFiles(root: PortablePath, {base}: {base: string}) {
-  const {stdout: branchStdout} = await execUtils.execvp(`git`, [`log`, `--format=`, `--name-only`, `${base}..HEAD`], {cwd: root, strict: true});
-  const branchFiles = branchStdout.split(/\r\n|\r|\n/).filter(file => file.length > 0).map(file => ppath.resolve(root, toPortablePath(file)));
```

Comment:
Computing changed files is something I write every so often and so far I've only ever used git-diff for tracked changes and git-ls-files for untracked files. Not sure what the third method is for but the correct semantics for those files would be "files changed during work" which is different than "files changed after work concluded".

Reply:
It is still failing since we're not using the newly built cli. You can verify locally that this indeed fixes it by running `yarn build:cli` followed by `yarn version check`.

Leaving CI running to see if those lines were tested.

Reply:
If you remove these lines Yarn won't detect at all the committed changes from master (because the following `git diff` is against `HEAD`, to detect the staged-but-not-committed changes). If you remove the `HEAD` from the diff, I think you'll also get the modified files when merging master.

Reply:
Wait why are we not comparing to `base`?

Reply:
Cf #427

Reply:
The test case added in that PR does not fail without the code change i.e. that test case is still passing on this branch. Could you create a test case that would fail now?

Reply:
I've tried but I can repro it, even after tweaking the test 😢

Well, at least if there was a problem it was causing false negatives rather than false positives, so it should reappear someday and we can add a better test 🤞